spec/amp-actions-and-events.md

```diff
@@ -182,20 +182,37 @@ event.index</pre></td>
   </tr>
 </table>
 
-### amp-selector
+### amp-sidebar
 <table>
   <tr>
     <th width="25%">Event</th>
     <th width="35%">Description</th>
     <th width="40%">Data</th>
   </tr>
   <tr>
-    <td><code>select</code></td>
-    <td>Fired when the user manually selects an option.</td>
-    <td><pre>// The option attribute
-// value of
-// the selected element.
-event.targetOption</pre></td>
+    <td><code>sidebarOpen</code></td>
+    <td>Fired when sidebar is fully opened after transition has ended.</td>
+    <td>None</td>
+  </tr>
+  <tr>
+    <td><code>sidebarClose</code></td>
+    <td>Fired when sidebar is fully closed after transition has ended.</td>
+    <td>None</td>
+  </tr>
+</table>
+
+### amp-carousel[type="slides"]
```

Comment:
This should already exist ?

Reply:
It's GitHub being bad at showing diffs. Nothing is deleted, just added sidebar above selector.

Reply:
Just GitHub being bad at showing diff. Nothing is deleted, just added sidebar above selector, diff shows up this way.